vita3k/kernel/src/kernel.cpp

```diff
@@ -55,8 +53,9 @@ Ptr<Ptr<void>> get_thread_tls_addr(KernelState &kernel, MemState &mem, SceUID th
 
         *address.get(mem) = new_tls;
         slot_to_address.insert(SlotToAddress::value_type(key, address));
-        return address;
     }
+    *(address.get(mem)) = 0;
```

Comment:
Why this?

Reply:
cause "Ar nosurge Plus - Ode to an Unborn Star" [PCSE00707] expect pointer to 0 on first run of sceKernelGetTLSAddr(43)
It was a fast hack. Going to rewrite it properly in a few days

Reply:
I rewrote it properly.
@frangarcj i've seen your last commit in get_thread_tls_addr What you were trying to do? what game you were trying to run? Sorry, if i look rude, my english is not good enough. I want to know, on what game i can run regression test after this further changes.